### PR TITLE
eslint the node code

### PR DESCRIPTION
**Pull request by peterbe**
Some basic testing using `eslint` of the Node code. 
This should leave the built-in `eslint`, that create-react-app does, alone. 
I dare not upgrade the version of eslint yet because we're going to soon upgrade to create-react-app 4.0 and that'll increment the version of `eslint` it wants to install.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  env: {
+    browser: false,
+    commonjs: true,
+    es6: true,
+  },
+  extends: ["eslint:recommended", "plugin:node/recommended"],
+  globals: {
+    Atomics: "readonly",
```
Comment (fiji-flo) [inline .eslintrc.js:9]: Out of curiosity: where do these come from?

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  env: {
+    browser: false,
+    commonjs: true,
+    es6: true,
```
Comment (fiji-flo) [inline .eslintrc.js:5]:
```suggestion
    es2020: true,
```

Comment (fiji-flo) [reply .eslintrc.js:5]: To be in sync with the `parserOptions`.
But for consistency with the node version we should use `es2019` until we move to node 14.

**Review by fiji-flo — CHANGES_REQUESTED** [review]
Everything else is good.

File: .eslintrc.js (hunk 1 above)
Comment (peterbe) [reply .eslintrc.js:9]: To be honest, it came from the `eslint --init` command I think.

**Review by fiji-flo — APPROVED** [review]

**Pull request merged**